core/server/services/auth/session/store.js

```diff
@@ -0,0 +1,48 @@
+const {Store} = require('express-session');
+
+module.exports = class SessionStore extends Store {
+    constructor(SessionModel) {
+        super();
+        this.SessionModel = SessionModel;
+    }
+
+    destroy(sid, callback) {
+        this.SessionModel
+            .findOne({session_id: sid})
```

Comment:
The base model destroy impl fetches and destroys the session by default.

e.g.
```
this.SessionModel.destroy()
   .catch(() => {
      catch 404 and call callback(null)
   })
```

You just need to allow fetching by `session_id`.

Reply:
Hmm - maybe I can kinda use it, seems like duplicate code, but i got to clean up the base model a bit so :smile:

Reply:
yeah you have to :)